recipe/meta.yaml

```
@@ -37,10 +40,13 @@ test:
     - test -f $PREFIX/lib/libpopt.dylib  # [osx]
 
 about:
-  home: http://rpm5.org/
+  home: https://rpm5.org/
```

```suggestion
  home: https://rpm.org/
```
If you remove `ftp.` from the source url `http://ftp.rpm.org/` you will get the home url.

---

Updated to ftp.rpm.org for source and rpm.org for home. Thanks!